content/sensu-core/0.29/reference/checks.md

```diff
@@ -737,22 +737,22 @@ example      | {{< highlight shell >}}"output": "i-424242"{{< /highlight >}}
 
 
 [?]:  #
-[1]:  ../clients
+[1]:  ../clients/
 [2]:  https://en.wikipedia.org/wiki/Standard_streams
 [3]:  https://www.nagios.org/
 [4]:  #check-results
-[5]:  ../../overview/architecture#event-processor
-[6]:  ../server
+[5]:  ../../overview/architecture/#event-processor/
```

Comment:
From testing this locally it appears that following anchor links with trailing slashes (e.g. `#event-processor/`) does not take us to the intended anchor